### PR TITLE
Update to newer k8s 1.25.6 as vendored dependency

### PR DESCRIPTION
**Pull request by dims**
Signed-off-by: Davanum Srinivas <davanum@gmail.com>

*Issue #, if available:*

*Description of changes:*

We are on a old version of k8s, let us please switch to a newer version.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by nckturner — APPROVED** [review]
Thanks!

**Comment by dims** [comment]
thanks @nckturner merging now.

File: main.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ import (
 	v1 "k8s.io/client-go/informers/core/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
-	"k8s.io/klog"
+	"k8s.io/klog/v2"
```
Comment (orsenthil) [inline main.go:39]: Nice! :)

**Review by orsenthil — APPROVED** [review]
LGTM.

**Pull request merged**